lms/templates/main.html

```diff
@@ -161,39 +161,39 @@
   <%block name="js_extra"/>
   <script type="text/javascript" src="${static.url('js/vendor/noreferrer.js')}" charset="utf-8"></script>
   <script type="text/javascript" src="${static.url('js/utils/navigation.js')}" charset="utf-8"></script>
+
+  <!-- Performance beacon for onload times -->
+  % if settings.FEATURES.get('ENABLE_ONLOAD_BEACON', False):
+  <script>
+    (function () {
+      var sample_rate = ${settings.ONLOAD_BEACON_SAMPLE_RATE};
+      var roll = Math.floor(Math.random() * 100)/100;
+      var onloadBeaconSent = false;
+
+      if(roll < sample_rate){
+        $(window).load(function() {
+          setTimeout(function(){
+            var t = window.performance.timing;
+
+            var data = {
+              event: "onload",
+              value: t.loadEventEnd - t.navigationStart,
+              page: window.location.href,
+            };
+
+            if (!onloadBeaconSent) {
+              $.ajax({method: "POST", url: "/performance", data: data});
+            }
+            onloadBeaconSent = true;
+          }, 0);
+        });
+      }
+    }());
+  </script>
+  % endif
```

Comment:
@clytwynec since you’ve added this script, could you check that everything is fine and still working please?

I don’t see why not, but surely there was a reason why it was outside of the `html` and `body` tags.